samples/DataSourceProvider/DataSourceProvider.Test/SampleProviderTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using TerraformPluginDotNet;
+using TerraformPluginDotNet.ResourceProvider;
+using TerraformPluginDotNet.Testing;
+using static Tfplugin5.Provider;
+
+namespace DataSourceProvider.Test;
+
+[TestFixture(Category = "Functional", Explicit = true)]
+public class SampleProviderTest
```

Comment:
Could you rename this to `SampleDataSourceProviderTest`, to match the `SampleDataSourceProvider` it's testing?